a10_neutron_lbaas/a10_config.py

```diff
@@ -28,7 +28,31 @@
 
 class A10Config(object):
 
-    def __init__(self, config_dir=None):
+    def __init__(self, config_dir=None, config=None):
+        if config is not None:
+            self._config = config
+            self._load_config()
+            return
+
+        self._config_dir = self._find_config_dir(config_dir)
+        self._config_path = os.path.join(self._config_dir, "config.py")
+
+        real_sys_path = sys.path
+        sys.path = [self._config_dir]
+        try:
+            try:
+                sys.modules.pop('config', None)
+                import config
+                self._config = config
+            except ImportError:
```

Comment:
`except ImportError` eats the import errors when trying to import `a10_neutron_lbaas.plumbing_hooks` to use the `VThunderPlumbingHooks`.